src/views/Landing.vue

```diff
@@ -53,7 +53,17 @@ export default {
   computed: {
     ...mapGetters(["account", "teamName"])
   },
+  beforeMount() {
+    const userName = this.$route.query.username || this.account?.name;
+
+    if (!userName) {
+      return;
+    }
+
+    this.setAccount({ name: userName });
```

Comment:
@tobiaschielens, wouldn't it be better if we add a star like we did with guests ?